completions/README.md

```diff
@@ -0,0 +1,5 @@
+# Shell completion scripts
+
+Please refer to each folder about how to install completion for `rmw` to your shell
+
+Feel free to contribute by opening PR providing completions to shells you like.
```

Comment:
This line can be removed, as we already have a contributing doc and mention contributing in the website.